<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cfg_Enviroment</class>
 <widget class="QWidget" name="cfg_Enviroment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxFont">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>471</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>471</width>
     <height>70</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>471</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Font</string>
   </property>
   <widget class="QLabel" name="labelSize">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>50</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Size:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFontSize">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelFont">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Family:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFontFamily">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>210</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxColorScheme">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>471</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>471</width>
     <height>241</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>471</width>
     <height>241</height>
    </size>
   </property>
   <property name="title">
    <string>Color Scheme</string>
   </property>
   <widget class="QListWidget" name="listWidgetColorScheme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>320</width>
      <height>210</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>100</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="buttonSchemeLayout">
     <item>
      <widget class="QPushButton" name="buttonNew">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
